### PR TITLE
Add color option for bookmark

### PR DESCRIPTION
**Pull request by 1v9**

### CODE REVIEW

**Comment by stevenjoezhang** [comment]
Which one is better?
```yml
color: FFF
#or
color: "#FFF"
```

https://github.com/theme-next/hexo-theme-next/blob/c970d79fc412f30a41d0ce7397306b87c30529c8/_config.yml#L370-L371
https://github.com/theme-next/hexo-theme-next/blob/c970d79fc412f30a41d0ce7397306b87c30529c8/_config.yml#L716

**Comment by 1v9** [comment]
`rating.color` is not belong to stylus.

```yml
color: "#8a2be2" # hex
color: blueviolet # html
color: rgb(138, 43, 226) # rgb
color: hsl(271, 76%, 53%) # hsl
```
Exclude hex value, other colors could be single or double quoted.

**Comment by 1v9** [comment]
https://www.w3.org/TR/css-color-3/#colorunits

**Comment by 1v9** [comment]
https://github.com/hexojs/hexo/blob/c2e4672e86a573d528a24b96fab63977172c0d64/lib/plugins/helper/tagcloud.js#L231-L249
https://github.com/hexojs/hexo/blob/c2e4672e86a573d528a24b96fab63977172c0d64/lib/plugins/helper/tagcloud.js#L180-L185
https://github.com/hexojs/hexo/blob/c2e4672e86a573d528a24b96fab63977172c0d64/lib/plugins/helper/tagcloud.js#L213-L226

File: source/css/_common/components/header/bookmark.styl [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
   top: -10px;
   transition: .3s;
 
   +tablet-mobile() {
-    display: none !important;
+    display: none;
+  }
+  &::before {
+    content: "\f02e";
+    font-family: $font-family-icons;
```
Comment (1v9) [inline source/css/_common/components/header/bookmark.styl]: here needs color

Comment (stevenjoezhang) [reply source/css/_common/components/header/bookmark.styl]: Why

Comment (1v9) [reply source/css/_common/components/header/bookmark.styl]: if not, hover color is `#222` not my custom color

**Pull request merged**